Dockerfile

```diff
@@ -20,16 +11,49 @@ RUN apk add --update --no-cache \
     giflib-dev \
     pango-dev
 
-# Install dependencies
+WORKDIR /src
+
 COPY package*.json ./
-RUN npm install
+RUN npm ci --no-audit
+
+# -- CHECK STAGE --------------------------------
+
+FROM base AS check
```

Comment:
Why `check`? 

That is, any reason why this is preferable to `dev`?

Reply:
Because this stage is related to checks, such as linting and unit tests.

Reply:
Hmm, i agree then.